man/adoc/bpftrace.adoc

```diff
@@ -1633,6 +1633,16 @@ bpftrace doesn't read past the length of the shortest string.
 
 The use of the `==` and `!=` operators is recommended over calling `strncmp` directly.
 
+=== strcontains
+
+.variants
+* `int64 strcontains(const char *haystack, const char *needle)`
+
+`strcontains` compares whether the string haystack contains the string needle.
+If needle is contained `0` is returned, else a non-zero value is returned.
```

Comment:
Looking at this now, why return 0 when the substring is found? I'd expect a function called `strcontains` returns true if it succeeds (so that you can test for the positive case, i.e. `if (strcontains(s1, s2)) ...`